src/pynwb/testing/mock/file.py

```diff
@@ -10,7 +10,7 @@
 def mock_NWBFile(
     session_description: str = 'session_description',
     identifier: Optional[str] = None,
-    session_start_time: datetime = datetime(1970, 1, 1, tzinfo=tzlocal()),
+    session_start_time: datetime = datetime(1970, 1, 1, 12, tzinfo=tzutc()),
```

Comment:
It seems from the discussion at the end of #2013 that using 0 hour and UTC time is OK, so I suggest using that, which is more standard and recognizable.
```suggestion
    session_start_time: datetime = datetime(1970, 1, 1, tzinfo=tzutc()),
```